### PR TITLE
Add paid

### PR DESCRIPTION
**Pull request by TheLillyleaf**

### CODE REVIEW

File: Payment/Card/index.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface Card {
 	limit: Amount
 	details?: Card.Details
 }
+
```
Comment (TheLillyleaf) [inline Payment/Card/index.ts:10]: remove

File: Payment/Expense/Paid.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { isly } from "isly"
+
+export interface Paid {
+	created: isoly.Date
+	issuer: string
+}
+
+export namespace Paid {
+	export const type = isly.object<Paid>({ created: isly.fromIs("date", isoly.Date.is), issuer: isly.string() })
+
```
Comment (TheLillyleaf) [inline Payment/Expense/Paid.ts:11]: remove

File: Receipt/index.spec.ts [hunk 3]
```diff
@@ -1,8 +1,4 @@
-import * as fs from "fs/promises"
-import { Blob, File } from "web-file-polyfill"
 import * as model from "../index"
```
Comment (TheLillyleaf) [inline Receipt/index.spec.ts:1]: use correct import

**Review by TheLillyleaf — COMMENTED** [review]
self check

File: Payment/index.ts [hunk 4]
```diff
@@ -25,6 +25,9 @@ export namespace Payment {
 	export const Creatable = PaymentCreatable
 	export type Expense = PaymentExpense
 	export const Expense = PaymentExpense
+	export namespace Expense {
+		export type Paid = PaymentExpense.Paid
+	}
```
[the remark this answers is not in the document]
Comment (EliasEriksson) [reply Payment/index.ts:30]: looking closer it looks like the type for  `issuefab.Payment.Card.Creatable` and `issuefab.Payment.PrePaid.Creatable` probably also are inaccessible

File: Purchase/index.spec.ts [hunk 5]
```diff
@@ -1,4 +1,3 @@
-import * as fs from "fs/promises"
 import * as model from "../index"
```
Comment (EliasEriksson) [inline Purchase/index.spec.ts:1]: update to new import

File: Purchase/index.ts [hunk 6]
```diff
@@ -1,6 +1,5 @@
 import * as cryptly from "cryptly"
```
Comment (EliasEriksson) [inline Purchase/index.ts]: update to new import

File: Receipt/index.ts [hunk 7]
```diff
@@ -1,6 +1,5 @@
 import * as cryptly from "cryptly"
```
Comment (EliasEriksson) [inline Receipt/index.ts]: update to new import

File: Report/Expense/index.spec.ts [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+import { issuefab } from "../../index"
+globalThis.Blob = Blob
+globalThis.FormData = FormData
+globalThis.File = File
+
+describe("ReportExpense", () => {
+	const file = { file: new File([new Uint8Array([97])], "file") }
+
+	it("is", () => {
+		expect(issuefab.Report.Expense.type.is(file)).toEqual(true)
```
Comment (EliasEriksson) [inline Report/Expense/index.spec.ts:12]: add one negative test

File: Report/Receipt/index.spec.ts [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+import { issuefab } from "../../index"
+globalThis.Blob = Blob
+globalThis.FormData = FormData
+globalThis.File = File
+
+describe("ReportReceipt", () => {
+	const file = { file: new File([new Uint8Array([97])], "file") }
+
+	it("is", () => {
+		expect(issuefab.Report.Receipt.type.is(file)).toEqual(true)
```
Comment (EliasEriksson) [inline Report/Receipt/index.spec.ts:12]: add one negative test

**Review by EliasEriksson — CHANGES_REQUESTED** [review]
double check to see if some type exports are missing. add them if they are missing. 
add a spec file for all new files and make sure the type exports are working there.

**Review by EliasEriksson — APPROVED** [review]

**Pull request merged**